qbft/spectest/tests/commit/current_round.go

```diff
@@ -20,7 +20,7 @@ func CurrentRound() tests.SpecTest {
 	return &tests.MsgProcessingSpecTest{
 		Name:          "commit current round",
 		Pre:           pre,
-		PostRoot:      "c3e7db86655d01a58ae7a54e4c181c1626014adf43c30d694785874e30c005b9",
+		PostRoot:      "ed3df7cffdc6fbbd989e58ece5327cd25c131fc76713b3cd1da6ba29cbf25568",
```

Comment:
@MatheusFranco99
Just that you know.
If you delete the PostRoot the test will pass.

When I started here I discovered some tests that were broken/incorrect.
So currently the convention I am using is to delete the PostRoot when I know the test is vetted.

I doubt that you reviewed all the tests here so what you did is perfectly fine

Reply:
Oh, nice to know that.
I'll follow this convention from now on, thanks!